autoload/lsp/ui/vim/documentation.vim

```diff
@@ -118,11 +118,14 @@ function! s:show_documentation(event) abort
     if l:right
         let l:line = a:event['row'] + 1
         let l:col = a:event['col'] + a:event['width'] + 1 + (a:event['scrollbar'] ? 1 : 0)
+        if l:col > &columns - 2
+            let l:col -= winwidth(0) / 2
```

Comment:
do we need float2nr here? we have seem multiple bugs on float values before.

Reply:
> can you explain what do you mean by minwidth and minheight? Do you also mean we need to add maxwidth and maxheight?

Yes, popup_create have attribute minwidth/minheight. This possibly can avoid either this issue.

Reply:
That sounds good.

Reply:
Hmm, minwidth and maxwidth may noy be good. It is not possible to calculate proper height and width. So I'll merge this workaround.